openassessment/xblock/test/test_submission.py

```diff
@@ -125,7 +125,7 @@ def test_closed_submissions(self, xblock):
     def test_prompt_line_breaks(self, xblock):
```

Comment:
@dmitry-viskov I would not modify the existing prompts. Our goal is to add your feature while supporting plaintext prompts. The unmodified defaults work in plaintext mode. Please revert `defaults.py` too.

This test should fail until your feature is backwards compatible.

Reply:
For now I think that `defaults.py` should be modified anyway because default prompt should be adapted for the HTML editor (I may assume that HTML editor should be default editor). I suggest to return to this question when backwards compatible question will be more clear